apps/angular/src/app/features/anime/anime-management/anime-form/anime-form.component.html

```diff
@@ -0,0 +1,126 @@
+<form [formGroup]="formGroup" (submit)="handleSubmit()" class="anime-form">
+
+    <div class="anime-form-section">
+        <mat-form-field 
+            *ngIf="formGroup.controls.englishTitle as englishTitle" 
+            class="anime-form-section__field">
+            <mat-label>English title</mat-label>
+            <input matInput [formControl]="englishTitle" />
+        </mat-form-field>
+    
+        <mat-form-field 
+            *ngIf="formGroup.controls.japaneseTitle as japaneseTitle"
+            class="anime-form-section__field">
+            <mat-label>Japanese title</mat-label>
+            <input matInput [formControl]="japaneseTitle" />
+        </mat-form-field>
+    </div>
+
+    <mat-form-field 
+        *ngIf="formGroup.controls.description as description"
+        class="anime-form-section__field">
+        <mat-label>Description</mat-label>
+        <textarea matInput rows="10" [formControl]="description"></textarea>
+    </mat-form-field>
+    
+    <div class="anime-form-section">
+        <mat-form-field 
+            *ngIf="formGroup.controls.type as type"
+            class="anime-form-section__field">
+            <mat-label>Type</mat-label>
+            <mat-select [formControl]="type">
+                <mat-option *ngFor="let type of animeTypes; trackBy trackByIndex" [value]="type">{{type}}</mat-option>
+            </mat-select>
+        </mat-form-field>
+    
+        <mat-form-field 
+            *ngIf="formGroup.controls.status as status"
+            class="anime-form-section__field">
+            <mat-label>Status</mat-label>
+            <mat-select [formControl]="status">
+                <mat-option *ngFor="let status of animeStatuses; trackBy trackByIndex" [value]="status">{{status}}</mat-option>
+            </mat-select>
+        </mat-form-field>
+    </div>
+
+    <div class="anime-form-section">
+        <mat-form-field 
+            *ngIf="formGroup.controls.ageRating as rating"
+            class="anime-form-section__field">
+            <mat-label>Rating</mat-label>
+            <mat-select [formControl]="rating">
+                <mat-option *ngFor="let rating of animeRatings; trackBy trackByIndex" [value]="rating">{{rating}}</mat-option>
```

Comment:
It's better to avoid variable names shadowing (we have two 'rating' variables here). It decreases code readability.
By the way, *ngIf looks redundant to me, we can inline it:
```suggestion
            class="anime-form-section__field">
            <mat-label>Rating</mat-label>
            <mat-select [formControl]="formGroup.controls.ageRating">
                <mat-option *ngFor="let rating of animeRatings; trackBy trackByIndex" [value]="rating">{{rating}}</mat-option>
```

Reply:
We have similar issue in other places, let's fix this everywhere.